themes/vuetober/src/components/inputs/BTextInputWithValidation.vue

```diff
@@ -4,6 +4,7 @@
     :name="$attrs.name"
     :rules="rules"
     v-slot="{ valid, errors }"
+    mode="eager"
```

Comment:
Neviem co toto robi, ale ked to fixlo tak fajn 🤷‍♂

Reply:
https://logaretm.github.io/vee-validate/guide/interaction-and-ux.html